buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -110,6 +110,14 @@ class ClusterConfiguration {
         return seedNode.transportUri()
     }
 
+    /**
+     * A closure to call which returns a manually supplied list of unicast seed hosts.
+     */
+    @Input
+    Closure<List<String>> manualSeedUris = {
```

Comment:
How about `otherUnicastHostAddresses` (they're not really URIs, and "seed" isn't quite the right word either)